modules/tree/main/index.coffee

```diff
@@ -82,22 +123,20 @@ class Tree
     @options = hx.merge.defined {
       hideDisabledButtons: false
       animate: true
-      renderer: (elem, data) -> hx.select(elem).html(data)
+      renderer: (elem, data) -> hx.select(elem).html(data.name || data)
```

Comment:
I figured adding this to the default renderer made sense as there's pretty much no point in having a tree that only has one level and to get more levels you need an object...